test/issue121-test.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-octal */
+// vim: expandtab:ts=2:sw=2
+
+const
+  assertions = require('./assertions'),
+  childProcess = require('./child-process').childProcess,
+  os = require('os'),
+  rimraf = require('rimraf'),
+  testCases = [
+    { signal: 'SIGINT', expectExists: false },
+    { signal: 'SIGTERM', expectExists: true }
+  ];
+
+const isWindows = os.platform() === 'win32';
+
+let tfunc = it;
```

Comment:
You could use a const if you change this to a ternary operation:

```javascript
const tfunc = isWindows ? xit : it;
```

Reply:
done